src/Form/input-classifiers.js

```diff
@@ -55,7 +58,7 @@ const getLargestMeaningfulContainer = (el, form) => {
 
     const inputsInScope = parentElement.querySelectorAll(FORM_ELS_SELECTOR)
     // To avoid noise, ensure that our input is the only in scope
-    if (inputsInScope.length === 1) {
+    if (inputsInScope.length <= 1) {
```

Comment:
With the previous conditional, if the parent already contained more than 1, the logic fell apart.

Reply:
The comments suggest this function only finds a containing node with one input child. I don't think that's the case is it?

The code here looks if there's 1 element and recurses the function call getting the containing node until:

- The element is the form
- The element is the top of the DOM tree
- Or the element has other inputs within the node.

Also in the updated change, surely this will always return true unless the selector doesn't match aka `el` is a buttton, radio etc.

Reply:
We decided the logic here was correct and just improved the var name. Other fixes made this work etc :)